### ISSUE TITLE
Laravel 5.3 Support

### ISSUE THREAD

**Issue by tylergets** [nilportugues/laravel5-jsonapi#102]
Release notes are located here:
https://laravel.com/docs/5.3/releases

Is anybody currently working on this?

**Comment by nojacko** [comment]
@tylergets Is 5.3 an issue? Doesn't look to me like any major changes. Would assume it works. I as because I'm trying to pick a JSON-API library for a new Laravel project... wanna pick the right one.

**Comment by mspiderv** [comment]
Im working on this.

**Comment by mpash** [comment]
@mspiderv Do you have an estimated time that this will be compatible with 5.3? Can we just pull in what you have and change the package requirements from 5.2 to 5.3 since there doesn't appear to be anything that will break? Thanks!

**Comment by mspiderv** [comment]
@mpash I dont know why tests failed. Cant fix this now. If you can help, do it.

**Comment by Stubbs** [comment]
I've fixed the tests, see my comment on https://github.com/nilportugues/laravel5-jsonapi/pull/108, but it breaks BC with Laravel 5.2

**Comment by mpash** [comment]
Yeah I'd love to help. I'm in the middle of a project at work but it makes sense to help out as part of the project

**Comment by mpash** [comment]
@mspiderv I've tried getting this up to spec, but I've hit a wall. I think this is beyond my abilities. Sorry.

**Comment by YanDatsyuk** [comment]
I got the error "Laravel version 5.3.26 is not supported. Please use the 5.1"

**Comment by mpash** [comment]
@nilportugues Any idea on when you'll be updating this or are we in the Abandonware stages?

**Comment by r-moiseev** [comment]
Trying to install on 5.3

So far i have issue with Passport that requires psr-http-message-bridge 1.0 but this package requires 0.1

**Comment by Luisetelo** [comment]
I also installed it on Laravel 5.3 and got the same error thant @YanDatsyuk when I try to consume the API.

I'm interested in this project, hope it wasn't deprecated yet.

**Comment by mspiderv** [comment]
Laravel 5.4 released :)

**Comment by sriramgk** [comment]
I tried on Laravel 5.4 :(. "Laravel version 5.4.18 is not supported. Please use the 5.1 for the time being".

**Comment by ghost** [comment]
@srigmca13 try `composer require nilportugues/laravel5-jsonapi dev-master`
Packagist may not have propagated yet. Use this trick to keep up-to-date.

**Comment by sriramgk** [comment]
@mpash I tried `composer require nilportugues/laravel5-jsonapi dev-master`

```
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package nilportugues/laravel5-jsonapi could not be found in any version, there may be a typo in the package name.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://getcomposer.org/doc/04-schema.md#minimum-stability> for more details.

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.

Installation failed, reverting ./composer.json to its original content.
```

I also tried without `dev-master` and getting the same error. Is anything I have to do with the composer config?

**Comment by sriramgk** [comment]
@mpash Is this correct one?

 `composer require nilportugues/laravel5-json-api dev-master`

**Comment by ghost** [comment]
@srigmca13 the format is `dev-{branch-name}`

**Comment by ghost** [comment]
@srigmca13 https://getcomposer.org/doc/articles/aliases.md

**Comment by pascal08** [comment]
`dev-master` is still not compatible with the L5.4 release. Any idea on when this will be made compatible? If you need help, please let me know. :)
